qml/pages/EditButton.qml

```diff
@@ -857,10 +857,11 @@ Dialog {
             id: openFileComponent
             OpenDialog {
                 title: qsTr("Select Soundfile")
-                path: _fm.getHome()
+                path: settings.value("lastFolder", _fm.getHome())
                 filter: ["*.mp3", "*.wav", "*.ogg", "*.flac", "*.m4a", "*.oga", "*.wma"]
                 onFileOpen: {
                     mfile = path;
+                    settings.setValue("lastFolder", _fm.getPath(path));
```

Comment:
Well I thought the same, but if you pass a path including the filename to the opendialog it will show an empty view. The fmhelper's getPath will strip out the filename and returns the path only.